why-polars-for-data-cleaning/index.qmd

```diff
@@ -0,0 +1,192 @@
+
+*Which tool should we use for processing data in our
+software and what tool should we recommend users to use when
+processing needs to happen outside of our software?*
+
+## Decision drivers
+
+::: content-hidden
+List some reasons for why we need to make this decision and what things
+have arisen that impact work.
+:::
+
+-   Can be installed and used in Python.
+-   Is designed and built to be fast for processing data.
+-   Has minimal dependencies, so can be installed quickly.
+-   Is flexible to reading different data storage formats (e.g. SQL
+    databases, Parquet, CSV).
+-   Can write to different storage formats (as with above).
+
+## Considered options
+
+::: content-hidden
+List and describe some of the options, as well as some of the benefits
+and drawbacks for each option.
+:::
+
+In addition to our requirements defined by our [principles](https://design.seedcase-project.org/software/principles) and our requirements listed above, these are the options we've considered:
+
+-   [DuckDB](https://www.duckdb.org/)
+-   [Pandas](https://pandas.pydata.org/)
+-   [Polars](https://pola.rs/)
+
+### DuckDB
+
+DuckDB is an in-process online analytical process (OLAP) database tool
+that has API extensions in many languages including Python. It is a very
+young tool, version 1.0.0 was released in 2024, although the first
+version was released in 2019. The developers have set it up as a
+not-for-profit foundation in the Netherlands.
+
+::: columns
+::: column
+#### Benefits
+
+-   Although a very young project it is already in use by a number of
+    large companies and has a large community of users.
+    -   The code is open source, and the way it has been set up is
+        promising for it to remain that way.
+-   It is very fast, and can handle large data sets.
+-   Extensive documentation with user guide sections embedded.
+:::
+
+::: column
+#### Drawbacks
+
+-   To use it to its full potential in Python, users need to have some knowledge
+    of SQL.
+:::
+:::
+
+### Polars
+
+The Polars project started in 2020 and is based in the Netherlands. They
+have a relatively small team but seem to have a fast growing user base,
+and very enthusiastic developers. Polars is written in Rust and
+available for Python. The two main developers are described as enthusiastic about
+open source.
+
+::: columns
+::: column
+#### Benefits
+
+-   Enthusiastic user community, with a lot of support available.
+-   Supports several languages, including Python.
+-   Handles all common data formats.
+-   Written in Rust (a very fast, low level language), so it processes data very fast.
+-   Polars has an extensive user guide section.
+:::
+
+::: column
+#### Drawbacks
+
+-   Relatively young project, so may not have all the features of more
+    established tools.
+-   Because it is still new, it isn't as widely used and doesn't have
+    too many learning resources available yet.
+-   No traditional documentation section, everything is in the user
+    guide.
```

Comment:
```suggestion
```

Hmm, based on your comment, I wouldn't suggest putting this under the drawbacks since the documentation is there on the navbar. As the text is now, it reads as if we couldn't find the documentation.